templates/vue.webapp/src/components/Tech.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div>
+    <ul v-for="technology in technologies" v-bind:key="technology.name" class="technologies">
+      <li>
+        <b>{{technology.name}}</b>: {{technology.details}}
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+
+export default {
+  name: 'Tech',
+  data() {
+    return {
+      technologies: []
+    };
+  },
+  mounted() {
+    axios
+      .get(`${process.env.VUE_APP_API_URL}/api/technologies`)
+      .then(response => (this.technologies = response.data));
```

Comment:
It is an unusual wrapping of a function body to `( ... )`. Why not `{ ... }`?

Reply:
I followed [official documentation](https://ru.vuejs.org/v2/cookbook/using-axios-to-consume-apis.html)

```javascript
new Vue({
  el: '#app',
  data() {
    return {
      info: null
    };
  },
  mounted() {
    axios
      .get('https://api.coindesk.com/v1/bpi/currentprice.json')
      .then(response => (this.info = response));
  }
});
```

Reply:
It should also be possible to leave the parentheses out completely in this case since they wrap only a single statement.